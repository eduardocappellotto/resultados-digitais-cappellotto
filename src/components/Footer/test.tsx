import { screen } from '@testing-library/react'

import { Footer } from '.'
import { renderWithTheme } from '@/utils/tests/helpers'

describe('<Footer />', () => {
  it('should render the footer', () => {
    const { container } = renderWithTheme(<Footer />)
    const copyright = `© ${new Date().getFullYear()} Resultados Digitais`

    expect(
      screen.getByRole('link', { name: /política de privacidade/i })
    ).toBeInTheDocument()
    expect(screen.getByText(copyright)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
