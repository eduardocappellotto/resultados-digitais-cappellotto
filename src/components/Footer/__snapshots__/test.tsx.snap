// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render the footer 1`] = `
.c2 {
  max-width: 128rem;
  width: 100%;
  margin: 0 auto;
  padding: 0;
}

.c4 {
  color: #FFFFFF;
  line-height: 1.7;
  font-weight: 400;
  font-size: 1.4rem;
}

.c0 {
  background-color: #FFFFFF;
  padding: 1.7rem 0;
}

.c0 .c1 {
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.c0 * {
  font-size: 1.4rem;
}

.c0 .c3 {
  color: #636E7C;
}

.c0 a {
  font-weight: 700;
  text-decoration: underline;
  color: #636E7C;
}

@media (max-width: 768px) {
  .c2 {
    padding: 0 2.4rem;
  }
}

@media (min-width: 768px) {
  .c0 .c1 {
    gap: 1.6rem;
    justify-content: center;
  }
}

<footer
  class="c0"
>
  <div
    class="c1 c2"
  >
    <a
      href="https://legal.rdstation.com/pt/privacy-policy/"
      target="_blank"
    >
      Política de Privacidade
    </a>
    <p
      class="c3 c4"
    >
      © 
      2024
       Resultados Digitais
    </p>
  </div>
</footer>
`;
